_commands/tools/dd.md

```diff
@@ -0,0 +1,88 @@
+---
+---
+
+dd
+--
+
+`dd` is a utility that can be used to convert and copy files in a number of ways. This can be used for backing up/restoring entire hard drives or partions, converting data formats, converting the case of a file, and more.
+
+~~~ bash
+$ dd if=[inputFile] of=[outputFile] [options]
+0+1 records in
+0+1 records out
+6 bytes copied, 0.000101046 s, 59.4 kB/s
+~~~
+
+<!--more-->
+
+### A Word of Caution
```

Comment:
Let's put this above any example. And in some large warning... I'm not quite sure how to do that right in Markdown, but I know this website uses Bootstrap so there's gotta be some easy CSS to throw in somehow.